<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateTimerSequenceDialog</class>
 <widget class="QDialog" name="CreateTimerSequenceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KitchenAlert - Create timer </string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>15</y>
     <width>731</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Alert text:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="timer1AlertText"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QSpinBox" name="Timer1HoursSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>h</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timer1minutesSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>min</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="timer1SecondsSpinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>s</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="standardButtons">
         <set>QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateTimerSequenceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>719</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>562</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
